Makefile

```
@@ -57,13 +57,26 @@ endif
 # Core Commands
 #-------------------------------------------------------
 
-.PHONY: help
+.PHONY: help ci
+
+ci:
```

checkmake (0.3.2)

[warning] Target body for "ci" exceeds allowed length of 5 lines (9).

(maxbodylength)